boards/arm/mimxrt685_evk/board.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2020,  NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_BOARD_H
+#define __INC_BOARD_H
+
+/*!< Board System oscillator settling time in us */
+#define BOARD_SYSOSC_SETTLING_US 260U
+/*!< Board xtal32K frequency in Hz */
+#define BOARD_XTAL32K_CLK_HZ 32768U
```

Comment:
i don't see where this definition is used. If it's not used, can we remove it?

Reply:
https://github.com/NXPmicro/zephyr/blob/MXRT600_Update_Core_Speed/soc/arm/nxp_imx/rt6xx/soc.c#L123-L125